### DISCUSSION TITLE
How to merge into a Homebrew formulae

### DISCUSSION THREAD

**Discussion by ninjacatz** [Homebrew/discussions#3639] [Everyday usage]
Hello!

I would like to merge this commit:
https://github.com/mupen64plus/mupen64plus-core/pull/970
With the homebrew formula `mupen64plus`
But I'm not even sure where to start.
Can I somehow use this file?
https://github.com/Homebrew/formulae.brew.sh/blob/1cfcb8d65ec784231d3de9f62122e509d08d1672/_data/formula/mupen64plus.json#L22

Could anyone give me a quick rundown of how to do this?

**Comment by carlocab** [comment]
The formula will need a `patch` block that applies the merged upstream commit at https://github.com/mupen64plus/mupen64plus-core/commit/5a5ffc0c78b0e255f54b15d2480f617d5cbc33ec.patch?full_index=1.

You can look at other formulae in the [homebrew-core](https://github.com/Homebrew/homebrew-core) repository for examples of how to do this; there are many.

Once you've done that, please open a pull request at homebrew-core. Here's a [guide](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request) for opening a pull request for Homebrew.

Note: it seems like you might have `HOMEBREW_INSTALL_FROM_API` set. You will need to unset this and tap homebrew-core (with `brew tap homebrew/core`) in order to make changes to the formula.
